.releaserc.yaml

```diff
@@ -1,6 +1,6 @@
 preset: conventionalcommits
 branches:
-  - name: rc
+  - name: main
```

Comment:
Did you re-stencil this? Not sure why this got changed, but we definitely don't want this.

Reply:
I think an earlier version of `stencil-base` was used

Reply:
I'll just re-run w/ frozen-lockfile